firstvoices/backend/rules.py

```diff
@@ -0,0 +1,5 @@
+import rules
+
+from backend.permissions.predicates.base import is_superadmin
+
+rules.add_perm("views.has_custom_order_access", is_superadmin)
```

Comment:
As the recalculate action is not tied to a model, I have opted for re-adding the superadmin rule. Additionally the recalculate view works the way the old recalculate preview view did before it was re-factored into a model view set. I am open to re-factoring the recalculate view as well if the team would prefer.